testutil/e2e/e2eProviderConfigs/lavaProvider9.yml

```diff
@@ -1,7 +1,7 @@
 endpoints:
   - api-interface: tendermintrpc
     chain-id: LAV1
-    network-address: 127.0.0.1:2264
+    network-address: 127.0.0.1:2274
```

Comment:
was there an issue with the original port? (just wanna know)

Reply:
This was because of the issue in the SDK, where both the Tendermint and Rest were required for lava over lava. This PR fix this problem https://github.com/lavanet/lava-sdk/pull/74